public/js/splitjs/splitjs.directive.js

```diff
@@ -0,0 +1,20 @@
+'use strict'; 
+
+module.exports =
+angular
+.module('diSplitJs.directive', [])
+.directive('split-area'), function(){
+    var directive; 
+    console.log('in split-area directive');
+    directive = {
+        restrict: 'E',
+        replace: true,
+        controller: 'diSplitJs',
+        template: '<div id="test-directive"><h3>Hello Directive</h3></div>',
+       // link: function($scope, $elem, attr){
+            
+      //  }
+    }
+    
+    return directive; 
+}
```

Comment:
Got it. I made the changes on the sub branch on feature_SplitPanes_Sub1. Should I just merge the changes of the sub branch to this branch?